src/inner_op/deformation.rs

```diff
@@ -229,19 +257,26 @@ pub fn new(parameters: &RawParameters, ctx: &dyn Context) -> Result<Op, Error> {
         ));
     }
 
-    let grid_file_name = params.text("grids")?;
-    let buf = ctx.get_blob(&grid_file_name)?;
-
-    let grid = Grid::gravsoft(&buf)?;
-    let n = grid.bands;
-    if n != 3 {
-        return Err(Error::Unexpected {
-            message: "Bad dimensionality of deformation model grid".to_string(),
-            expected: "3".to_string(),
-            found: n.to_string(),
-        });
+    let grid_names = params.text("grids")?;
```

Comment:
Use `param.texts("grids")` here (note plural `texts`), and leave out the `split(',')` below. Also update `GAMUT` to expect `texts` rather than `text`

(this comment applies to both deformation, datum shift, and geoid grids)

Reply:
There wasn't a `texts()` method in parse_parameters so I've added it as 

```rust
pub fn texts(&self, key: &str) -> Result<&Vec<String>, Error> {
        if let Some(value) = self.texts.get(key) {
            return Ok(value);
        }
        Err(Error::MissingParam(key.to_string()))
    }
```

Reply:
Oops - sorry: I have evidently missed that when I introduced `texts` in 49384de. Thanks for repairing that!